test/test_palo_alto_collectors.py

```diff
@@ -64,21 +66,20 @@ def test_get_all_interfaces(username,
                 'ip',
                 'id',
                 'addr']
-    assert df_all.columns.to_list() == expected
+    assert set(df_all.columns.to_list()) == set(expected)
```

Comment:
Is "set" being used to eliminate duplicate entries?  If so, it may be better to resolve this in the collectors or parsers so we don't have issues with the NM database.

Reply:
When it was comparing columns with expected columns '==' was also matching the order of the columns. Using set removes the need for them to be in order.